### ISSUE TITLE
Can I compile code without `cljs-runtime` dependent in watch mode?

### ISSUE THREAD

**Issue by idhowardgj94** [thheller/shadow-cljs#1132]
As title, 
I'm trying to build my electron app in clojurescript using shadow-cljs,

Here come's one problem: preload script seems didn't support load code dynamically, which lead the preload script broken in watch / compile mode (run successfully n production mode)

seems preload is just a one time loading task, and didn't need `shadow-cljs / clojurescript`'s repl support, 
Are there any chance that I can disable cljs-runtime part of code?

or I just need to write vannila js code in preload part of code.

**Comment by thheller** [comment]
No, that is not supported. You can just manually trigger a `release` build for the preload and inject that.

**Comment by idhowardgj94** [comment]
Ok I see, thx
